app/controllers/users_controller.rb

```diff
@@ -10,14 +10,6 @@ def create
 
     if @user.save
       token = SessionService.send_magic_link_email!(@user.email_address)
-      # NOTE: this is purely to allow us to display the link in the footer
-      # REMOVE when we have emails actually being sent
-      identifier = @user.sign_in_identifier(token)
-      @debug_info = {
```

Comment:
Shoud we also remove all references to the `show_debug_info` feature flag?

Reply:
yes, good idea - but I'll do that as a separate PR